src/pretix/base/models/event.py

```diff
@@ -376,8 +380,25 @@ def annotated(cls, qs, channel, voucher=None):
                 'quotas',
                 to_attr='active_quotas',
                 queryset=quota_base_qs.annotate(
-                    active_items=Subquery(sq_active_item, output_field=models.TextField()),
-                    active_variations=Subquery(sq_active_variation, output_field=models.TextField()),
+                    active_items=Subquery(
+                        sq_active_item.order_by().values_list('quotas__pk').annotate(
+                            items=GroupConcat('pk', delimiter=',')
+                        ).values('items'),
+                        output_field=models.TextField()
+                    ),
+                    active_variations=Subquery(
+                        sq_active_variation.order_by().values_list('quotas__pk').annotate(
+                            items=GroupConcat('pk', delimiter=',')
+                        ).values('items'),
+                        output_field=models.TextField()),
+                    has_active_items_with_waitinglist=Exists(
+                        sq_active_item.filter(allow_waitinglist=True),
+                        output_field=models.TextField()
```

Comment:
What is the reasoning behind using an output_field other than the default BooleanField? Or maybe just an artefact from the previous solution using GroupConcat-Query?

```suggestion
```